ExpandableListView/build.gradle.kts

```diff
@@ -0,0 +1,72 @@
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.jetbrains.kotlin.android)
+    alias(libs.plugins.compose.compiler)
+    id(libs.versions.mavenpublish.get())
+}
+
+afterEvaluate {
+    publishing {
+        publications {
+            // Creates a Maven publication called "release".
+            create<MavenPublication>(libs.versions.release.get()) {
+                from(components[libs.versions.release.get()])
+                groupId = libs.versions.group.id.get()
+                artifactId = libs.versions.artifact.id.get()
+                version = libs.versions.version.get()
+            }
+        }
+    }
+}
+
+android {
+    namespace = "com.simform.expandablelistview"
+    compileSdk = 35
+
+    defaultConfig {
+        minSdk = 24
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Need to update this to Java 11 for jitpack

Reply:
Create a separate PR to develop branch for this fix

Reply:
Updated.